src/modules/data/blocks/ticket/sagas.js

```diff
@@ -908,8 +925,9 @@ export function* syncTicketSaleEndWithEventStart( prevStartDate, clientId ){
 		// If initial end and current end are the same, the RSVP has not been modified
 		const isNotManuallyEdited = yield call( [ tempEndMoment, 'isSame' ], endMoment, 'minute' );
 		const isSyncedToEventStart = yield call( [ tempEndMoment, 'isSame' ], prevEventStartMoment, 'minute' );
+		const isEvent = yield call( isTribeEventPostType );
 
-		if ( isNotManuallyEdited && isSyncedToEventStart ) {
+		if ( isEvent && window.tribe.events && isNotManuallyEdited && isSyncedToEventStart ) {
```

Comment:
Was this not working before? Or was this running in non-event post types? The `window.tribe.events` and the `try-catch` are redundant.

Reply:
This was running on non-event post types. I thought the catch would be useful to see unexpected errors, this would be an expected bypass

Reply:
done